lib/ansible/modules/storage/netapp/na_ontap_gather_facts.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/python
+
+# (c) 2018 Piotr Olczak <piotr.olczak@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: na_ontap_gather_facts
+author: Piotr Olczak (polczak@redhat.com)
+extends_documentation_fragment:
+    - netapp.na_ontap
+short_description: NetApp information gatherer
+description:
+    - This module allows you to gather various information about ONTAP configuration
+version_added: "2.7"
+requirements:
+    - netapp_lib
+options:
+    state:
+        description:
+            - Returns "info"
+        default: "info"
+        required: false
+        choices: ['info']
```

Comment:
we would have liked to only use present and absent for state.

But since it's optional, no big deal.

Reply:
Ok, I'll keep it like this.